### ISSUE TITLE
Convert bid page to swap style

### ISSUE THREAD

**Issue by luziusmeisser** [Frankencoin-ZCHF/frankencoin-dapp#95]
Today, the bid page looks like this:

![image](https://github.com/user-attachments/assets/9747001a-8761-47e7-b266-efd1121bddd9)

I think it would be quite intuitive to the user if the input amount (the user balance) has its own input field, making it more similar to the swap page, which currently looks like this:

![image](https://github.com/user-attachments/assets/049d914b-a089-4327-b40d-4773bb7d6d87)

This is a low prio item, but it might be worthwhile doing once we have reworked all the main pages.

**Comment by luziusmeisser** [comment]
(might be easier said than done because the price keeps changing)

**Comment by samclassix** [comment]
> (might be easier said than done because the price keeps changing)

yeah, thats actually a bit tricky. Not only the price is changing, the bid will be in the collateral size and with "floating points".

**Comment by luziusmeisser** [comment]
outdated
